src/bp-templates/bp-nouveau/includes/template-tags.php

```diff
@@ -1333,6 +1337,10 @@ function bp_nouveau_get_nav_count() {
 		$count = bp_media_get_total_group_media_count();
 	} elseif ( 'groups' === $bp_nouveau->displayed_nav && bp_is_active( 'media' ) && bp_is_group_albums_support_enabled() && 'albums' === $nav_item->slug ) {
 		$count = bp_media_get_total_group_album_count();
+	} elseif ( 'groups' === $bp_nouveau->displayed_nav && 'courses' === $nav_item->slug ) {
```

Comment:
@shohelbuddyboss what will happen if someone change `courses` to something else via `apply_filters( 'bp_ld_sync/courses_group_tab_slug', 'courses' )` this filter.

Reply:
Edited and applied the same filters on this sting, so if anyone changes, it will change also here.
Code I wrote, `$coursesSlug     = apply_filters( 'bp_ld_sync/courses_group_tab_slug', 'courses' );`